pkg/boot/kexec/purg_load.go

```diff
@@ -14,36 +14,42 @@ import (
 	"github.com/u-root/u-root/pkg/boot/align"
 )
 
-// PurgeLoad loads an elf file at the desired address.
+// PurgeLoad loads an ELF (which must be one of the two purgatory from the
```

Comment:
which must be one of the two purgatories

Reply:
done